bot/zephyr.py

```diff
@@ -443,6 +444,29 @@ def main(cfg):
 
         print("Done")
 
+    if 'postrun' in cfg:
+        print("Running postrun tasks ...")
+
+        postrun = cfg['postrun']
+
+        # Run script
+        if 'script' in postrun:
+            script = postrun['script']
+            if os.path.isfile(script):
+                script_proc = subprocess.Popen(shlex.split(script),
+                                               shell=True)
+                if script_proc.wait():
+                    print("Failed to run post script")
+
+        # Run post run script
+        if 'cleanup' in postrun:
```

Comment:
[1] here

Reply:
I was going to use this routine to upload the test result to Zephyr TestRail site and reset the VM. I need to more think about it...   :)